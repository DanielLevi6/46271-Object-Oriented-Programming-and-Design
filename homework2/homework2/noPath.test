# noPath test file

# Create a graph
CreateGraph myGraph

# Add nodes
CreateNode node1 5
CreateNode node2 10
CreateNode node3 1

AddNode myGraph node1
AddNode myGraph node2
AddNode myGraph node3

# Add an edge
AddEdge myGraph node1 node2
AddEdge myGraph node1 node3

# Find a path
FindPath myGraph node2 -> node3
