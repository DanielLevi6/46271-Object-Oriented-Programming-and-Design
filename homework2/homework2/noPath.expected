# noPath test file

# Create a graph
created graph myGraph

# Add nodes
created node node1 with cost 5
created node node2 with cost 10
created node node3 with cost 1

added node node1 to myGraph
added node node2 to myGraph
added node node3 to myGraph

# Add an edge
added edge from node1 to node2 in myGraph
added edge from node1 to node3 in myGraph

# Find a path
no path found in myGraph
